README.md

```diff
@@ -1,21 +1,35 @@
 # Mobile-First
 Project Name
-Portfolio 
+Simple mobile form of mu portfolio
 
 Built With
-Html
-css 
+html
+css
+figma
+
 Getting Started
+
 To get a local copy up and running follow these simple example steps.
-copy repository url
-clone the repo on any command line using git clone (url)
+
+copy the url of the repository 
+clone the repository in a folder using comand line 
 Install
- run 
+Use
+Authors
+👤 Donald Akite
 
+GitHub: @quavo19
+Twitter: @donaldakite
+LinkedIn: donaldakite
 
-Author
-👤 Akite Donald
+Feel free to check the issues page.
 
-GitHub: @quavo19
 Show your support
-Give a ⭐️ if you like this project!
\ No newline at end of file
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Please you should use hashtags for the titles in your `readme.md` file to make it more readable instead of using raw text. :pray:. As requested by the previous code reviewer you should follow the README [template](https://github.com/microverseinc/readme-template) which contains all sections.

  - HIT: You can copy all the sections in the template and modify it with your project details. :+1: